app/src/main/kotlin/com/hippo/ehviewer/ui/main/FabLayout.kt

```diff
@@ -131,7 +124,7 @@ fun FabLayout(
 
     val secondaryFab by delegateSnapshotUpdate {
         record { buildFab(builder) }
-        transform { onEachLatest { state.waitCollapse() } }
+        transform { onEachLatest { state.collapse(MutatePriority.PreventUserInput) } }
```

Comment:
What's this for?
Secondary fab will recompose when a new item is selected, resulting in the selection mode to be exited.

Reply:
Secondary fab should not recompose when a new item is selected, let's fix it

Reply:
For why we force it collapse, see https://github.com/FooIbar/EhViewer/pull/1288/commits/7e1bc58711e444bc7779baf024681637a62da73f